plugins/carbon_black_defense/plugin.spec.yaml

```diff
@@ -523,184 +135,388 @@ types:
+      description: "Device policy id"
+      required: false
+    device_id:
+      title: "Device Id"
       type: integer
-    longDescription:
-      title: Long Description
-      type: string
-    threatIndicators:
-      title: Threat Indicators
-      type: '[]string'
-    securityEventCode:
-      title: Security Event Code
-      type: string
-    deviceDetails:
-      title: Device Details
-      type: deviceDetails
-    orgDetails:
-      title: Org Details
-      type: orgDetails
-    eventType:
-      title: Event Type
-      type: string
-    syslogLevel:
-      title: Syslog Level
-      type: object
-    netFlow:
-      title: Net Flow
-      type: netFlow
-    shortDescription:
-      title: Short Description
-      type: string
-    targetApp:
-      title: Target Hash
-      type: hash
+      description: "Device id"
+      required: false
+    event_network_inbound:
+      title: "Event Network Inbound"
+      type: boolean
+      description: "Event network inbound"
+      required: false
+    process_guid:
+      title: "Process GUID"
+      type: string
+      description: "Process GUID"
+      required: false
+    device_location:
+      title: "Device Location"
+      type: string
+      description: "Device location"
+      required: false
+    netconn_local_port:
+      title: "Netconn Local Port"
+      type: integer
+      description: "Netconn local port"
+      required: false
+    device_os_version:
+      title: "Device Os Version"
+      type: string
+      description: "Device os version"
+      required: false
+    org_id:
+      title: "Org Id"
+      type: string
+      description: "Org id"
+      required: false
+    device_name:
+      title: "Device Name"
+      type: string
+      description: "Device name"
+      required: false
+    backend_timestamp:
+      title: "Backend Timestamp"
+      type: string
+      description: "Backend timestamp"
+      required: false
+    parent_pid:
+      title: "Parent PID"
+      type: integer
+      description: "Parent PID"
+      required: false
+    device_policy:
+      title: "Device Policy"
+      type: string
+      description: "Device policy"
+      required: false
+    parent_reputation:
+      title: "Parent Reputation"
+      type: string
+      description: "Parent reputation"
+      required: false
 connection:
   url:
     title: URL
     type: string
     description: API URL
-    default: https://api.confer.net
+    default: https://defense.conferdeploy.net
     required: true
+    example: https://example.com
   api_key:
     title: API Key
     type: credential_secret_key
     description: API key
     required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  org_key:
+    title: Organization Key
+    type: string
+    description: Organization key
+    required: true
+    example: 1ABCDEFGH
   connector:
     title: Connector
     type: string
     description: Connector ID
     required: true
+    example: 1ABCD23333
```

Comment:
Is this in the format (aka length, kinds of characters) that one would expect for a connector ID? If so I'm happy to leave this example